### ISSUE TITLE
[BUG] Click assignment for hotkeys

### ISSUE THREAD

**Issue by maboroshin** [Moosync/Moosync#299]
**Describe the bug**
Assigning a hotkey to the left click makes it difficult to use the app.

**To Reproduce**
1. Go hotkeys in the settings.
2. Left-click Keybinds column for "Reload page". Once more click.
3. ```Mouse0``` is assigned.
4. Close the settings page.
5. Left click now reloads the page. 
6. I can't re-open the settings window.

**Expected behavior**
Nothing can be assigned to ```Mouse0``` and ```Mouse2```. 

Note: In my environment, The right mouse click for left-handed mice is also ```Mouse0```.

- OS: Win10
- Version 10.2.0

**Comment by Ovenoboyo** [comment]
Yup it makes sense, will disable assignment for mouse0 and mouse2
